nightly-benchmark/assertions.py

```diff
@@ -0,0 +1,48 @@
+import yaml
+
+import dask
+import distributed
+
+def check_scheduler_is_cythonized(client: distributed.Client):
+    # TODO is there a less hacky way to do this?
+    path = client.run_on_scheduler(lambda: distributed.scheduler.__file__)
+    if not path.endswith(".so"):
```

Comment:
This was added in PR ( https://github.com/dask/distributed/pull/4764 ), but is not in a release yet (though this may be a non-issue if we are just building with `main` from source)

```suggestion
    if not distributed.scheduler.COMPILED:
```